repos/system_upgrade/el7toel8/actors/bindupdate/libraries/updates.py

```diff
@@ -0,0 +1,86 @@
+from leapp.libraries.common import isccfg
+from leapp.libraries.stdlib import run
+
+# Callback for walk function
+callbacks = {
+    'dnssec-lookaside': isccfg.ModifyState.callback_comment_out,
+}
+
+
+def paths_from_issues(issues):
+    """ Extract paths from list of BindConfigIssuesModel """
+    paths = []
+    for issue in issues:
+        paths.append(issue.path)
+    return paths
+
+
+def parser_file(parser, path):
+    for cfg in parser.FILES_TO_CHECK:
+        if cfg.path == path:
+            return cfg
+    return None
+
+
+def debug_log(log, text):
+    if log is not None:
```

Comment:
just ```if log:```?

Reply:
Removed that altogether and used global logger.